zeta-cpi/tests/zeta_cpi.ts

```diff
@@ -31,7 +33,9 @@ describe("zeta-cpi", () => {
     throw new Error("ANCHOR_PROVIDER_URL is not defined");
   }
   const connection = new anchor.web3.Connection(url, utils.defaultCommitment());
-  const provider = new anchor.Provider(
+  const asset = assets.Asset.BTC;
```

Comment:
Is this intentionally restricted to just BTC?

Reply:
I think the only issue here is that `pythOracle` gets set to the BTC oracle but should be grabbed when making each transaction

Reply:
nvm this is a test pls ignore